libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -491,7 +491,15 @@ void AP_MotorsMatrix::setup_motors(motor_frame_class frame_class, motor_frame_ty
 
     switch (frame_class) {
 
-        case MOTOR_FRAME_QUAD:
+        case MOTOR_FRAME_TRI:
```

Comment:
I think, combine indentation with other CASE

Reply:
not sure what you mean